payload-files/nextjs-14_2_1/route-handlers/_api_test-error--event.json

```diff
@@ -4,14 +4,9 @@
     "event_id": "[[ID1]]",
     "sdk": {
       "name": "sentry.javascript.nextjs",
-      "version": "7.110.1"
+      "version": "8.0.0-beta.1"
     },
-    "sent_at": "[[ISODateString]]",
-    "trace": {
-      "environment": "qa",
-      "public_key": "[[publicKey]]",
-      "trace_id": "[[ID2]]"
-    }
```

Comment:
The trace headers are missing in v8 but should be fixed with https://github.com/getsentry/sentry-javascript/pull/11628